spec/middlewares/raise_error_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "spec_helper"
+
+describe BaseCRM::Middlewares::RaiseError do
+  before { @raise_error = BaseCRM::Middlewares::RaiseError.new }
+
+  describe "on_complete" do
```

Comment:
use symbol instead of string to be consistent with other specs: `describe :on_complete do`